automatic/puppet-bolt/update.ps1

```diff
@@ -2,8 +2,8 @@ import-module au
 
 # No trailing slash
 # Order is important.  Most recent first
-$downloadURLs = @('https://downloads.puppet.com/windows/puppet6',
-                  'https://downloads.puppet.com/windows/puppet5')
+$downloadURLs = @('https://downloads.puppet.com/windows/puppet-tools',
```

Comment:
The puppet-tools repo only goes back to 1.27.0. Are we okay with removing support for installing anything earlier?